collect_app/src/main/java/org/odk/collect/android/utilities/FileUtils.java

```diff
@@ -100,6 +103,39 @@
     private FileUtils() {
     }
 
+    public static void saveMediaFileFromUri(Uri uri, File destFile, Context context) {
```

Comment:
Could this maybe be `answerFile` or `instanceFile`? The `QuestionMediaManager` refers to these as "answer files". I'm frequently getting confused between files in the instance folder and files that come with the form.

Reply:
Ok, done.